kuma/dashboards/forms.py

```diff
@@ -6,7 +6,7 @@
 
 
 LANG_CHOICES = [('', _('All Locales'))] + settings.LANGUAGES
-
+PERIOD_CHOICES = (('', 'None'), ('hour','Hour'), ('day','Day'), ('week','Week'), ('month','30 days'))
```

Comment:
The human-readable option labels need the tower gettext_lazy `_` wrapper. Like the `LANG_CHOICES` tuple above.